base.requirements.txt

```diff
@@ -1,4 +1,5 @@
 ## Base requirements
+dojson==0.4.0
```

Comment:
Is that line ever run? We don't do `pip install .` or python setup.py build`, at least on Travis